proto/composerapis/astria/composer/v1alpha1/grpc_collector.proto

```diff
@@ -0,0 +1,18 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+import "google/protobuf/empty.proto";
+
+message RollupTransaction {
+  bytes rollup_id = 1;
+  bytes tx_bytes = 2;
+}
+
+message SubmitRollupTransactionsRequest {
+  repeated RollupTransaction rollup_txs = 1;
+}
+
+service GrpcCollectorService {
+  rpc SubmitRollupTransactions(SubmitRollupTransactionsRequest) returns (google.protobuf.Empty) {}
```

Comment:
I believe it would be a breaking change, at least based on how we have defined as strictly as possible (based on file). We should have an empty return object instead. 

Similar to how we use empty request objects for `GetGenesisInfoRequest` in sequencer API.